src/PostgREST/ApiRequest.hs

```diff
@@ -191,11 +191,19 @@ userApiRequest schema req reqBody =
     where lastWord = last $ T.split (=='.') key
 
   headerRange = if singular && method == "GET" then singletonRange 0 else rangeRequested hdrs
-  urlOffsetRange = rangeGeq . fromMaybe (0::Integer) $
-    readBSMaybe =<< join (lookup "offset" qParams)
-  urlRange = restrictRange
-    (readBSMaybe =<< join (lookup "limit" qParams))
-    urlOffsetRange
+  replaceLast x s = T.intercalate "." $ L.init (T.split (=='.') s) ++ [x]
```

Comment:
then the line would get longer since i would need an additional fromMaybe, not sure